doc/classes/Array.xml

```diff
@@ -25,6 +29,10 @@
 		array[1] = "Second";
 		GD.Print(array[1]); // Prints "Second"
 		GD.Print(array[^3]); // Prints "Second"
+
+		// This array can only contain integers.
+		// Attempting to add any other type will result in an error.
+		Godot.Collections.Array&lt;int&gt; typedArray = [1, 2, 3];
```

Comment:
Is this valid C# syntax, or do we need to go the long way and use `new Godot.Collections.Array` for the righthand side?

Reply:
We've been using it already, see https://github.com/godotengine/godot-docs/pull/10433. Since Godot 4.4, the minimum required .NET version is 8 (which defaults to C# 12) so this syntax is guaranteed to be supported.

Reply:
OK, I'll revert to the new syntax I used previously then.